src/legilogic_ethereum/ethereum_user.ml

```diff
@@ -619,6 +619,49 @@ let create_contract ~sender ~code ?gas_limit value =
 let call_function ~sender ~contract ~call ?gas_limit value =
   make_pre_transaction ~sender (Operation.CallFunction (contract, call)) ?gas_limit value
 
+let print_status_receipt : TransactionReceipt.t -> string =
+  fun tr -> (TokenAmount.to_string tr.status)
```

Comment:
Why go through a string?

Reply:
If you're going to translate it, shouldn't it be either to a `bool` or some other `type transaction_status = success | failure` ?

Reply:
String has been removed.